game/src/engine/asset/AssetService.cpp

```diff
@@ -5,7 +5,8 @@
 
 using namespace std;
 
-void AssetService::LoadMesh(const string &path, const string &name)
+void AssetService::LoadMesh(const string &path, const string &name,
+                            const string &textureName)
```

Comment:
I think we may want to have loading textures and loading meshes be separate, so you could just make a separate function for loading textures instead. In the future when we do UI stuff, we're gonna wanna be able to load _just_ textures without meshes, so then we wouldn't need to re-write any texture loading stuff. Also, at least while debugging, it's very possible we're gonna have meshes without textures.

So I think it makes it easier and more flexible for us in the future if Meshes and Textures aren't necessarily always tied together